### PR TITLE
Add labels for RHODS

### PR DESCRIPTION
**Pull request by chainlink**

### CODE REVIEW

**Review by nitinjainsj — COMMENTED** [review]
The same change would be needed on build pipeline images. The docker files are in a different dir. and the goreleaser/docker-build-pipeline.yml

File: goreleaser/docker.yml [hunk 1]
```diff
@@ -42,17 +42,23 @@ dockers:
       - "--build-arg=GO_VERSION={{ .Env.GOVERSION }}"
       - "--build-arg=LD_FLAGS={{ .Env.LD_FLAGS }}"
       - "--progress=plain"
+      - "--label=version={{.Version}}"
```
[the remark this answers is not in the document]
Comment (chainlink) [reply goreleaser/docker.yml:45]: this is what I got after running locally:

```
docker inspect -f {{.Config.Labels}} pachyderm/pachd:latest
map[name:Pachyderm release:v1.13.0-rc.1-SNAPSHOT-409606f27 vendor:Pachyderm version:v1.13.0-rc.1-SNAPSHOT-409606f27]
```

[the remark this answers is not in the document]
Comment (nitinjainsj) [reply goreleaser/docker.yml:45]: Oh this is great.

**Comment by chainlink** [comment]
@nitinjainsj ready for another pass

**Review by nitinjainsj — APPROVED** [review]
LGTM

**Pull request merged**